Sources/eppo/ConfigurationRequester.swift

```diff
@@ -32,7 +32,7 @@ class ConfigurationRequester {
             debugLogger?("Network request completed (\(String(format: "%.1f", networkDurationMs))ms, Data: \(data.count) bytes)")
         }
         
-        debugLogger?("Starting JSON parsing and configuration creation")
+        debugLogger?("Starting JSON parsing and configuration creation (parsing \(data.count) bytes)")
```

Comment:
If we discover that JSON parsing large data is the bottle neck, some ideas:

* Parse on background queue to unblock initialization
* Partially defer parsing: eager load user-provided flag keys, lazy load the others (on demand or in the background)